SAMD21/Rev B/RS485_OD/RS485_OD.ino

```diff
@@ -202,7 +205,7 @@ void read_MuxShield() {
   
   for (int h=0; h<(PDtimes_averaged); h++){
     mux_total = mux_total + readMux(active_vial);
-    serialEvent(1);
```

Comment:
That's because I changed the `serialEvent` function to not be based on waiting a set amount of time but until either the serial is done sending messages or a `!` char is found.